api/python/t4/snapshots.py

```diff
@@ -216,6 +216,21 @@ def as_dict(self):
         }
         return copy.deepcopy(ret)
 
+    @staticmethod
+    def from_local_path(path):
+        with open(path, 'rb') as file_to_hash:
+            hash_obj = {
+                'type': 'SHA256',
+                'value': hash_file(file_to_hash)
+            }
+
+        size = os.path.getsize(path)
+        physical_keys = [{
+            'type': PhysicalKeyType.LOCAL.name,
+            'path': os.path.abspath(path)
+        }]
+        return PackageEntry(physical_keys, size, hash_obj, {})
```

Comment:
Add a todo to ingest metadata here when https://github.com/quiltdata/t4/pull/28 is ready